dev-client/src/components/FeatureFlagControl.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+import {useState} from 'react';
+import {StyleSheet, View} from 'react-native';
+import {Divider, Switch} from 'react-native-paper';
+
+import {
+  Heading,
+  Text,
+} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {APP_CONFIG, MMKV} from 'terraso-mobile-client/config';
+
+export const FeatureFlagControlOrEmptyInProd = () => {
```

Comment:
I think it's fine to just call this component `FeatureFlagControl`, the use-case of hidden in prod is implicit imo.